internal/testplanet/planet_test.go

```diff
@@ -61,6 +62,19 @@ func TestBasic(t *testing.T) {
 	time.Sleep(time.Second)
 }
 
+func TestPlanet_Ping(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount:   1,
+		StorageNodeCount: 10,
+		UplinkCount:      0,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		time.Sleep(5 * time.Second)
```

Comment:
The sleep shouldn't be necessary anymore.

Reply:
i think this was a copy/paste but good to know